app/forms/webauthn_setup_form.rb

```diff
@@ -64,6 +67,12 @@ def name_is_unique
     @name_taken = true
   end
 
+  def validate_transports
+    return if transports.blank?
+    return if transports.difference(WebauthnConfiguration::VALID_TRANSPORTS).blank?
+    errors.add(:transports, :general, type: :invalid_transports)
```

Comment:
We should probably log something here for us to be notified. The specification makes some disclaimers about future-compatibility, though I don't know that we'd want to go as far as to allow _any_ value here.

>It is important for backwards compatibility that [client platforms](https://w3c.github.io/webauthn/#client-platform) and [Relying Parties](https://w3c.github.io/webauthn/#relying-party) handle unknown values.

https://w3c.github.io/webauthn/#sct-domstring-backwards-compatibility

Reply:
And maybe we don't want to block the user if we start getting unexpected `transports` values. We could either strip out the unrecognized ones, or set it to `nil` as an indication that we don't want to pass those hints in future authentications.

Reply:
then maybe we do `&` for intersection to limit to only known values

Reply:
Added logging in 62b0e19. Went with setting `nil` for any unrecognized as of 988762b.